static/templates/user_info_popover_content.hbs

```diff
@@ -67,15 +67,15 @@
             </a>
         </li>
         {{/if}}
-        <li>
-            <a href="#" class="update_status_text">
-                <i class="fa fa-comments" aria-hidden="true"></i>
-                {{#if status_text}}
-                    {{#tr this}}Edit status message{{/tr}}
-                {{else}}
-                    {{#tr this}}Set a status message{{/tr}}
-                {{/if}}
-            </a>
+        <li class="update_status_text" id="status_messages">
```

Comment:
The `id` should probably be singular, since it's only one message.

Reply:
should I write it edit_status_message